src/gui/pages_menu/KM_GUIMenuCampaign.pas

```diff
@@ -201,18 +201,14 @@ procedure TKMMenuCampaign.StartClick(Sender: TObject);
 
 procedure TKMMenuCampaign.AnimNodes(aTickCount: Cardinal);
 begin
-  if fCampaign <> nil then
+  if fCampaign = nil then Exit;
+  if Image_CampaignSubNode[fAnimNodeIndex].Visible then Exit;

```

Comment:
If fCampaign.Maps[fMapIndex].NodeCount is equal to MAX_CAMP_NODES it will crash here after animating the final node